k8s/staging.template.yaml

```diff
@@ -0,0 +1,165 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: eventapi
+  namespace: app
+spec:
+  selector:
+    matchLabels:
+      app: eventapi
+  template:
+    metadata:
+      labels:
+        app: eventapi
+    spec:
+      volumes:
+        - name: config
+          configMap:
+            name: eventapi-config
+            defaultMode: 420
+      containers:
+        - name: eventapi
+          image: ${IMAGE}
+          ports:
+            - name: http
+              containerPort: 3000
+              protocol: TCP
+            - name: metrics
+              containerPort: 9100
+              protocol: TCP
+            - name: health
+              containerPort: 9200
+              protocol: TCP
+          envFrom:
+            - secretRef:
+                name: eventapi-secret
+          env:
+            - name: API_K8S_POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: API_K8S_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+          resources:
+            limits:
+              cpu: "2"
+              memory: 3Gi
+            requests:
+              cpu: 300m
+              memory: 256Mi
```

Comment:
this is WAY too low, need to bump this to 2Gi

Reply:
i am okay with over committing cpu but memory over commit it very scary since we can run into oom here. This app uses a lot of memory.